docs/topics/html-and-forms.md

```diff
@@ -66,9 +66,10 @@ The following view demonstrates an example of using a serializer in a template f
 
         def post(self, request, pk):
             profile = get_object_or_404(Profile, pk=pk)
-            serializer = ProfileSerializer(profile)
+            serializer = ProfileSerializer(profile, data=request.POST)

```

Comment:
I think we can use [`request.data`](http://www.django-rest-framework.org/api-guide/requests/#data) for consistency.